src/Check/UsersBlockedNumberOne.php

```diff
@@ -58,10 +68,19 @@ public function getResultInfo() {}
    * {@inheritdoc}.
    */
   public function getResultPass() {
-    if ($this->registry->vendor == "pantheon") {
-      return $this->t('UID #1 is not blocked, but --vendor=pantheon exempts this test.');
+    // If the result is passed but the account is unblocked, the request must
+    // have come from inside the platform.
+    if ($this->status == 1) {
+      return $this->t('UID #1 is not blocked, but blocking UID #1 is not recommended on the Pantheon platform.');
```

Comment:
We shouldn't mention Pantheon here. How about "UID #1 is not blocked. Blocking this user eliminates a potential security risk."

Nothing on Pantheon makes it safer to leave UID #1 unblocked.

Reply:
I updated the text, the change was based on the ticket's observations that the change was unnecessary on Pantheon but I agree that blocking UID 1 is never a bad idea.